include/swift/AST/RawComment.h

```diff
@@ -38,7 +38,9 @@ struct SingleRawComment {
   SingleRawComment(StringRef RawText, unsigned StartColumn);
 
   SingleRawComment(const SingleRawComment &) = default;
-  SingleRawComment &operator=(const SingleRawComment &) = default;
+
+  // 'Range' has no copy assignment operator; therefore:
```

Comment:
That seems like a bug, actually. CharSourceRange should be a trivial type. Or is this about the `const`?

Reply:
'const' does seem to be the problem.